### PR TITLE
Safety first

### LINKED ISSUE #18: Don't dig into lava
When digging stairs

```
/Users/darius/minecraft/spoon/node_modules/mineflayer/lib/plugins/digging.js:88
      bot.blockAt(bot.entity.position).type === 9, !bot.entity.onGround); // only stationary water counts
                                      ^

TypeError: Cannot read property 'type' of null
```

### PR DESCRIPTION
**Pull request by LowWeiLin**
Make digging safe again
Fixes #18

### CODE REVIEW

File: src/tasks/tasks.js [hunk 1]
```diff
@@ -158,14 +181,21 @@ function digStairTask(steps: number, height?: number) {
...
-    // Dig down
-    return dig().then(() => sleep(1000))
-                .then(() => digWallTask(h))
-                // Move forward
-                .then(() => moveTo(center(bot.entity.position.offset(1,0,0))))
-                .then(() => sleep(100))
-                // Repeat
-                .then(() => digStairTask(steps-1, h));
+    var nextBlock = findSafeStandingBlockForward();
+    var nextPosition = bot.entity.position;
```
Comment (kanghj) [inline src/tasks/tasks.js:185]: don't need this line?

Comment (LowWeiLin) [reply src/tasks/tasks.js:185]: Need... I had some problems with flow where the nextPosition/block could be null, even though i checked it.

Comment (LowWeiLin) [reply src/tasks/tasks.js:185]: I think may be able to fix it though. @dariusf added some types

Comment (kanghj) [reply src/tasks/tasks.js:185]: Hmm, I don't get an error with flow though, after shifting the variable declaration into the `nextBlock != null` branch, what error am I supposed to get?

**Pull request merged**